app/src/main/java/org/literacyapp/authentication/AuthenticationActivity.java

```diff
@@ -72,6 +76,9 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_authentication);
 
+        screenBrightnessMode = DetectionHelper.getScreenBrightnessMode(getApplicationContext());
```

Comment:
What does "Detection" mean? Maybe something like ScreenHelper or BrightnessHelper would be a better name?